### DISCUSSION TITLE
Gists

### DISCUSSION THREAD

**Discussion by AdrianBinDC** [community/community#3784] [Mobile] [answered]
I would love to view gists within the GitHub app.

**Comment by amrehman** [comment, marked as answer]
Thanks a lot for your feedback!

I've added your note to our internal issue where we are tracking adding gists. If you have any other feedback or feature requests, please keep them coming!

**Reply by Angelk90** [reply]
@amrehman : More than two years have passed, is there any news?

**Reply by adriancuadrado** [reply]
@Angelk90 2 months and @amrehman didn't even answer. This is hopeless.

**Comment by kanition** [comment]
Still nothing in 2025
